bin/generate_commit

```diff
@@ -104,6 +104,28 @@ function _create_changelog() {
         --date=iso
 }
 
+function _create_readme(){
+
+    local title=$1
+    local summary=$2
+    
+    local tree=$(tree )
+
+    cat << EOF > README.md
+    ## $title
```

Comment:
Unfortunately this will add the indentation to generated file. The whole section should probably have 0 indent so the generated file doesn't look funny. In addition, start the first heading using `#`